recipes-connectivity/prplmesh/prplmesh/003_prplmesh_util_stop_use_hard_coded_wlan_iface_names.patch

```diff
@@ -0,0 +1,52 @@
+commit d090b1ad12036d6339746258c8b7be053f27395e
```

Comment:
This one was merged upstream.

Reply:
Used prplMesh commit will be updated in separate task https://jira.prplfoundation.org/browse/PPM-281